tests/gis_tests.cc

```diff
@@ -0,0 +1,84 @@
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include <string>
+#include "gis.h"
+
+using namespace std;
+
+using cyclus::GIS;
+
+TEST(GISTests, check_distance) {
+  GIS eiffel(48.858222, 2.2945);
```

Comment:
These objects should be created in a setup function and the tests can then use them as test fixtures (TEST_F).